x-pack/plugins/fleet/README.md

```diff
@@ -13,7 +13,7 @@
 
 ## Fleet Requirements
 
-Fleet needs to have Elasticsearch API keys enabled, and also to have TLS enabled on kibana, (if you want to run Kibana without TLS you can provide the following config flag `--xpack.fleet.agents.tlsCheckDisabled=false`)
+Fleet needs to have Elasticsearch API keys enabled, and also to have TLS enabled on Kibana, (if you want to run Kibana without TLS you can provide the following config flag `--xpack.fleet.agents.tlsCheckDisabled=false`)
```

Comment:
I think only the first part is true now `Fleet needs to have Elasticsearch API keys enabled`. The `tlsCheckDisabled` has been deprecated and it's possible to run Kibana without TLS in development

Reply:
Thanks @nchaulet - actually, could this whole section be reframed better within the `Development` section below?

Reply:
I've removed this bit for now. Maybe the first two sections (`Plugin` and `Fleet Requirements`) could get a closer look at, possibly at a later stage.